cmd/ocitools/generate.go

```diff
@@ -382,3 +406,15 @@ func setupLinuxNamespaces(g generate.Generator, needsNewUser bool, nsMaps map[st
 		g.AddOrReplaceLinuxNamespace(nsName, nsPath)
 	}
 }
+
+func onlyExportFlagSpecified(context *cli.Context) bool {
+	onlyExportFlags := []bool{
+		context.Bool("seccomp-only"),
+	}
+	for _, flag := range onlyExportFlags {
+		if flag {
+			return true
```

Comment:
What is this function about?  Why can't we just `context.Bool("seccomp-only")` instead of `onlyExportFlagSpecified`?